cogs/quote_commands.py

```diff
@@ -112,6 +100,24 @@ async def quote(self, ctx:utils.Context, messages:commands.Greedy[discord.Messag
                 embed.description = text
             # embed.set_footer(text=f"Quote ID {quote_id.upper()}")
             embed.timestamp = timestamp
+            return embed
+
+
+        
+        
+    @utils.group(invoke_without_command=True)
+    @commands.bot_has_permissions(send_messages=True, embed_links=True, add_reactions=True)
+    @commands.guild_only()
+    async def quote(self, ctx:utils.Context, messages:commands.Greedy[discord.Message]):
+        """
+        Quotes a user's message to the guild's quote channel.
+        """
+              # Make sure no subcommand is passed
+        if ctx.invoked_subcommand is not None:
+            return
+        embed = await startquote(self, ctx, messages)
```

Comment:
This doesn't account for errored invokes (no messages quoted; invalid messages; wrong authors; etc). This also needs to be `self.startquote`.